main.css

```diff
@@ -0,0 +1,358 @@
+    background-attachment: fixed;
+    background-size: cover;
+    margin: 0;
+    padding: 0;
+}
+
+#homePage{
+    display: grid;
+    grid-template-columns: 20vw 20vw 20vw 20vw 20vw;
+    grid-template-rows: auto;
+    grid-template-areas: 
+    "header header header header header"
+    "restlist map map map map "
+    "footer footer footer footer footer"
+}
+
+li{
+    border-radius: 5%;
+    max-width: 20vw;
+    padding: 3px;
+    text-align: center;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 20px;
+    height: auto;
+    width: auto;
+    border: 1px black solid;
+    margin: 10px;
+    background-color:white;
+    box-shadow: 2px 2px 2px black;
+}
+li:hover{
+    box-shadow: 4px 4px 4px black;
+}
+
+@media (max-width: 600px){
+    #yelp{
+        font-size: 50px !important;
+    }
+    #bars{
+        font-size: 15px !important;
+    }
+    li{
+        font-size:15px;
+    }
+}
+@media (max-width: 420px){
+    #yelp{
+        font-size: 30px !important;
```

Comment:
You should be very sparing with where you use the `!important` tag. You probably don't need it here if you put your query selectors at the bottom of your CSS file. Nice use of media queries to make your page responsive!